sandbox/grist/table.py

```diff
@@ -89,6 +89,9 @@ def lookupRecords(self, **field_value_pairs):
     ```
 
     See [RecordSet](#recordset) for useful properties offered by the returned object.
+    
+    See [CONTAINS](#contains) for an example utilizing `UserTable.lookupRecords` to find records where
+    a column contains the given value.
```

Comment:
The word "column" here is confusing -- these are records which have a field of type ReferenceList or ChoiceList, and CONTAINS allows finding those where such a "list field" contains the given value.

Reply:
updated in second commit